libs/adapters/tsconfig.json

```diff
@@ -1,11 +1,13 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "rootDir": "src",
-    "outDir": "build",
+    "rootDir": ".",
     "tsBuildInfoFile": "build/.tsbuildinfo",
     "strict": true
   },
   "include": ["src"],
-  "references": [{ "path": "../core" }, { "path": "../logging" }]
+  "references": [
```

Comment:
did you try without references here? not sure this is needed when not building

Reply:
TBH we could move `adapters/src/index.ts` to `/adapters/index.ts` and we wouldn't need the references. Not sure how you feel about moving that file tho.